CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Add the `languages` directory to the `.npmignore` whitelist.
```

Comment:
According to https://keepachangelog.com/en/1.0.0/ it's more a `fix` than `new feature`?

> Added for new features.
> Changed for changes in existing functionality.
> Deprecated for soon-to-be removed features.
> Removed for now removed features.
> Fixed for any bug fixes.
> Security in case of vulnerabilities.

Should we use headers to categorize changes @scarletfog ?

Reply:
```suggestion

### Fixed
- Add the `languages` directory to the `.npmignore` whitelist.
```

❓

Reply:
And IMO the message does not explain to end-user why we did that. What she can expect now?

Reply:
Yes, it's a fix. 
Yes, please add headers (I think we should update the changelog.md with a small snippet with all options)

And some additional info for this point would be nice, imagine you have to prepare release notes based on this file only.

Reply:
Will do